### PR TITLE
Update documentation for widget "page"

### PR DESCRIPTION
**Pull request by SirTobiIV**
add German description for page name and add description for page attribute "visible".

### CODE REVIEW

File: src/visu_config.xsd [hunk 1]
```diff
@@ -563,7 +564,12 @@
             <xsd:appinfo>level:expert</xsd:appinfo>
         </xsd:annotation>
     </xsd:attribute>
-    <xsd:attribute name="visible" type="xsd:boolean" />
+    <xsd:attribute name="visible" type="xsd:boolean">
+    	<xsd:annotation>
+        	<xsd:documentation xml:lang="en">Displays the widget (page name) which links to be page</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: I would suggest "Show a widget, which links to the page" I don't think that it is clear what the text in the brackets means, and I think we do not need it here.

File: src/visu_config.xsd [hunk 2]
```diff
@@ -563,7 +564,12 @@
...
+        	<xsd:documentation xml:lang="de">Zeigt ein Widget an (Seitenname), das auf die Seite verlinkt</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: According to the above suggestion: "Zeigt ein Widget an, das auf die Seite verlinkt"

Comment (ChristianMayer) [inline src/visu_config.xsd]: Oder: Zeigt ein Widget mit dem Seitennamen als Link auf die Seite.

File: src/visu_config.xsd [hunk 3]
```diff
@@ -566,8 +566,8 @@
     </xsd:attribute>
     <xsd:attribute name="visible" type="xsd:boolean">
     	<xsd:annotation>
-        	<xsd:documentation xml:lang="en">Displays the widget (page name) which links to be page</xsd:documentation>
-        	<xsd:documentation xml:lang="de">Zeigt ein Widget an (Seitenname), das auf die Seite verlinkt</xsd:documentation>
+        	<xsd:documentation xml:lang="en">Displays a widget with the page name being a link to be page</xsd:documentation>
```
Comment (peuter) [inline src/visu_config.xsd]: type: please change "to be page" to "to the page"

**Comment by SirTobiIV** [comment]
I ahve changed that according to your comments.

**Comment by peuter** [comment]
There is still a misspelled word in the english text, as I mentioned lately. If you fix that I would merge this pull request.

**Comment by SirTobiIV** [comment]
done - sorry overlooked that comment

**Pull request merged**